config/vaccines.yml

```diff
@@ -21,3 +21,8 @@
       product_codes:
         - system: 'http://hl7.org/fhir/sid/cvx'
           code: '212'
+    - product_name: 'AstraZeneca COVID-19 Vaccine (non-US)'
```

Comment:
This needs to be added to the [advanced filter options](https://github.com/SaraAlert/SaraAlert/blob/1a9755b25c6c87963b7480eadef3aa7c2573ed07/app/javascript/data/advancedFilterOptions.js#L329) as well. I feel like ideally those options would auto populate from this file so this wasn't an issue, but not sure if that is not easy to accomplish.

Reply:
See here for why the decision was made for the lists to be separate: https://github.com/SaraAlert/SaraAlert/pull/894#discussion_r634457717

Reply:
1611696